dockstore-common/src/test/java/io/dockstore/common/yaml/DockstoreYamlTest.java

```diff
@@ -182,4 +183,22 @@ public void testDifferentCaseForWorkflowSubclass() throws DockstoreYamlHelper.Do
         assertEquals(1, workflows.stream().filter(w -> "cwl".equals(w.getSubclass())).count());
     }
 
+
+    @Test
+    public void testIncorrectProperty()  {
+        // Replace:
+        // ...
+        // workflows:
+        //
+        // With incorrect:
+        // ...
+        // workflow:
+        final String content = DOCKSTORE12_YAML.replaceFirst("workflows", "workflow");
+        try {
+            DockstoreYamlHelper.readAsDockstoreYaml12(content);
```

Comment:
This test won't quite work if it throws some other exception or just doesn't throw an exception

Reply:
Fixed up test

Reply:
Won't this still "succeed" with some other exception, say an IOException?

Reply:
Now will catch any other exception and fail the test